src/Microsoft.OpenApi.OData.Reader/Vocabulary/Capabilities/UpdateRestrictionsType.cs

```diff
@@ -49,10 +49,10 @@ internal class UpdateRestrictionsType : IRecord
         public bool? DeltaUpdateSupported { get; private set; }
 
         /// <summary>
-        /// Gets the value indicating the HTTP Method (PUT or PATCH) for updating an entity. 
+        /// Gets the values indicating the HTTP Method (PUT and/or PATCH) for updating an entity. 
         /// If null, PATCH should be supported and PUT MAY be supported.
         /// </summary>
-        public HttpMethod? UpdateMethod { get; private set; }
+        public IList<HttpMethod> UpdateMethods { get; private set; }
```

Comment:
I believe the ask here is to make the enum flaggable with the [Flags] attribute. https://github.com/microsoft/OpenAPI.NET.OData/blob/e6efed827b5d5cb8748c8077bd835ae8c8bd97b8/src/Microsoft.OpenApi.OData.Reader/Vocabulary/Capabilities/UpdateRestrictionsType.cs#L17

As renaming the property would be breaking and would not match the spec name
https://github.com/oasis-tcs/odata-vocabularies/blob/a7850482cc83488d37c66c4e907f6a94406e1d05/vocabularies/Org.OData.Capabilities.V1.xml#L764C23-L764C33

Reply:
Thanks for the insightful heads up! I've updated as suggested.